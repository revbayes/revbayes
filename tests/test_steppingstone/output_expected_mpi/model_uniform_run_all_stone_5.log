Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21819.55	-21804.78	-14.7742	0.9469134	0.149238	0.0978573	0.0571549	0.112164	0.152448	0.146327	0.221183	0.415433	0.070276	0.134167	0.183518	0.196827	0.114955	0.149775	0.206353	0.174596	0.23403	0.101222	0.170293	0.409628	0.184965	0.129121	0.244273	0.0953506	0.0140349	0.00147779	0.0860618	0.224664	0.0352614	0.120263	0.215766	0.09339	0.174642	0.000797676	0.0861989	0.0473986	0.0823614	0.139179	0.105254	0.0270916	0.119986	0.24753	0.166137	0.0442397	0.336845	0.0569143	0.0354145	0.490379	0.0362074	0.126244	0.460192	0.990615	2.42295	0.342389	0.343305	0.0850852	0.229221	0.4425109	6.13865
10	-22029.5	-22011.43	-18.0746	0.7491524	0.250167	0.178306	0.0813557	0.188053	0.0580078	0.188536	0.213212	0.0301569	0.150738	0.057232	0.252699	0.167062	0.0526216	0.191279	0.371025	0.186912	0.207807	0.248695	0.144206	0.435165	0.193136	0.375526	0.265999	0.0580886	0.0663329	0.00371764	0.0289202	0.14413	0.0649193	0.0962232	0.202445	0.111662	0.167602	0.00338389	0.0657643	0.0450949	0.0355624	0.237428	0.181765	0.0452347	0.0798991	0.0925887	0.214741	0.071764	0.40851	0.0264251	0.0134783	0.45625	0.0235728	0.0847032	0.385827	0.941805	2.58766	0.250396	0.388641	0.09176	0.269203	0.4551531	6.4334
