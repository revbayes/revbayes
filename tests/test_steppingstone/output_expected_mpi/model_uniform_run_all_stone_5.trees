Iteration	Posterior	Likelihood	Prior	psi
0	-21819.55	-21804.78	-14.7742	(Galeopterus_variegatus[&index=23]:0.244273,(Tarsius_syrichta[&index=21]:0.184965,(((Pan_paniscus[&index=17]:0.234030,Hylobates_lar[&index=9]:0.070276)[&index=32]:0.093390,((Macaca_mulatta[&index=13]:0.114955,Chlorocebus_aethiops[&index=5]:0.152448)[&index=25]:0.014035,Colobus_guereza[&index=6]:0.146327)[&index=42]:0.247530)[&index=30]:0.120263,(Callicebus_donacophilus[&index=2]:0.097857,(Saimiri_sciureus[&index=20]:0.409628,(Cebus_albifrons[&index=3]:0.057155,Aotus_trivirgatus[&index=1]:0.149238)[&index=33]:0.174642)[&index=26]:0.001478)[&index=43]:0.166137)[&index=31]:0.215766)[&index=36]:0.047399,(Daubentonia_madagascariensis[&index=7]:0.221183,(((Otolemur_crassicaudatus[&index=16]:0.174596,Galago_senegalensis[&index=8]:0.415433)[&index=24]:0.095351,Nycticebus_coucang[&index=15]:0.206353)[&index=29]:0.035261,((Perodicticus_potto[&index=18]:0.101222,Loris_tardigradus[&index=12]:0.196827)[&index=37]:0.082361,(Lepilemur_hubbardorum[&index=11]:0.183518,(((Propithecus_coquereli[&index=19]:0.170293,Cheirogaleus_major[&index=4]:0.112164)[&index=40]:0.027092,(Lemur_catta[&index=10]:0.134167,Varecia_variegata_variegata[&index=22]:0.129121)[&index=38]:0.139179)[&index=27]:0.086062,Microcebus_murinus[&index=14]:0.149775)[&index=39]:0.105254)[&index=35]:0.086199)[&index=41]:0.119986)[&index=28]:0.224664)[&index=34]:0.000798)[&index=44];
10	-22029.5	-22011.43	-18.0746	((((((Perodicticus_potto[&index=18]:0.248695,((Galago_senegalensis[&index=8]:0.030157,Nycticebus_coucang[&index=15]:0.371025)[&index=24]:0.058089,Otolemur_crassicaudatus[&index=16]:0.186912)[&index=29]:0.064919)[&index=41]:0.079899,Loris_tardigradus[&index=12]:0.167062)[&index=37]:0.035562,Lepilemur_hubbardorum[&index=11]:0.252699)[&index=34]:0.003384,Daubentonia_madagascariensis[&index=7]:0.213212)[&index=35]:0.065764,((((Cebus_albifrons[&index=3]:0.081356,(Saimiri_sciureus[&index=20]:0.435165,Aotus_trivirgatus[&index=1]:0.250167)[&index=26]:0.003718)[&index=33]:0.167602,Callicebus_donacophilus[&index=2]:0.178306)[&index=43]:0.214741,(Hylobates_lar[&index=9]:0.150738,(((Chlorocebus_aethiops[&index=5]:0.058008,Colobus_guereza[&index=6]:0.188536)[&index=25]:0.066333,Macaca_mulatta[&index=13]:0.052622)[&index=42]:0.092589,Pan_paniscus[&index=17]:0.207807)[&index=32]:0.111662)[&index=30]:0.096223)[&index=31]:0.202445,Tarsius_syrichta[&index=21]:0.193136)[&index=36]:0.045095)[&index=28]:0.144130,Galeopterus_variegatus[&index=23]:0.265999,((Microcebus_murinus[&index=14]:0.191279,Propithecus_coquereli[&index=19]:0.144206)[&index=27]:0.028920,(Cheirogaleus_major[&index=4]:0.188053,(Varecia_variegata_variegata[&index=22]:0.375526,Lemur_catta[&index=10]:0.057232)[&index=38]:0.237428)[&index=40]:0.045235)[&index=39]:0.181765)[&index=44];
