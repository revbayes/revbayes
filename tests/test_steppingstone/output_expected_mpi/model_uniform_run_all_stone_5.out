state	power	likelihood
4	0.03125	-21855.4
5	0.03125	-21825.9
6	0.03125	-21785.4
7	0.03125	-21865.1
8	0.03125	-21911.6
9	0.03125	-21944.2
10	0.03125	-22011.4
